mms/metric.py

```diff
@@ -113,6 +129,23 @@ def start_recording(self):
                 with open(filename, 'a') as csvfile:
                     csvwriter = csv.writer(csvfile, delimiter=',')
                     csvwriter.writerow([utcnow, metric])
+            elif self.write_to == 'cloudwatch':
+                logger.info('Metric %s for last %s seconds is %f, writing to AWS CloudWatch...' %
+                    (self.name, self.interval_sec, metric))
+                try:
+                    self.client.put_metric_data(
+                        Namespace='mxnet-model-server',
```

Comment:
Can we have another component in the namespace for the model name?
This will allow users who run multiple models to be able and break the metrics per model.